CHANGELOG.md

```diff
@@ -1,6 +1,8 @@
 # PD Change Log
 
 ## v2.0.1
+### New Feature
+* Add the `Scatter Range` scheduler to balance regions with the specified key range
```

Comment:
regions -> Regions

Note: "Region" is a proper noun for the TiDB project, so please capitalize the first letter of "Region" to keep it consistent with its occurrences in other documents.

Reply:
got it. done.